aiocache/decorators.py

```diff
@@ -310,19 +309,19 @@ async def decorator(
     ):
         missing_keys = []
         partial = {}
-        keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
+        orig_keys, cache_keys, new_args, args_index = self.get_cache_keys(f, args, kwargs)
```

Comment:
Consider naming these as `in_keys` and `out_keys` for clarity.  Respectively they are the keys that are passed in to the callable, and the keys that are returned by the callable (`out_keys` are also the translated keys returned by `key_builder`).

If these `in_keys` are only used internally, then it might make sense to leave `get_cached_keys()` unmodified (i.e., revert it).  Then instead `in_keys` could be generated here from `_get_args_dict()`. `get_cache_keys()` is a public member function.

Reply:
> Consider naming these as `in_keys` and `out_keys` for clarity. Respectively they are the keys that are passed in to the callable, and the keys that are returned by the callable (`out_keys` are also the translated keys returned by `key_builder`).

This is under the assumption of [Interpretation 2](https://github.com/aio-libs/aiocache/pull/564#issuecomment-1380574858).